cdc/kv/etcd_test.go

```diff
@@ -471,10 +471,7 @@ func (s *etcdSuite) TestCreateChangefeed(c *check.C) {
 		SinkURI: "root@tcp(127.0.0.1:3306)/mysql",
 	}
 
-	err := s.client.CreateChangefeedInfo(ctx, detail, "bad.id👻")
```

Comment:
Why is this case deleted?

Reply:
https://github.com/pingcap/ticdc/pull/2054/files#diff-cc1de9ecd51350251a4837f8c67b15b653ed937eedf6cc61daebdf28644bb973L301
move the validation to `cmd/client_changefeed.go`, validate first
https://github.com/pingcap/ticdc/pull/2054/files#diff-75cf14b54b555126511c6140e14beb602a4c7a0ccec637a7996e9d2cf256c6dcR462